icu4c/source/common/rbbitblb.cpp

```diff
@@ -1386,6 +1380,7 @@ void RBBITableBuilder::exportTable(void *where) {
 
     table->fNumStates = fDStates->size();
     table->fDictCategoriesStart = fRB->fSetBuilder->getDictCategoriesStart();
+    table->fLookAheadResultsSize = fLASlotsInUse == ACCEPTING_UNCONDITIONAL ? 0 : fLASlotsInUse + 1;
```

Comment:
Would it be easier to read by the following?
table->fLookAheadResultsSize = (fLASlotsInUse == ACCEPTING_UNCONDITIONAL) ? 0 : (fLASlotsInUse + 1);